devolutions-gateway/Cargo.toml

```diff
@@ -64,7 +64,7 @@ tracing-appender = "0.2"
 # Async, futures…
 tokio = { version = "1.36", features = ["signal", "net", "io-util", "time", "rt", "rt-multi-thread", "sync", "macros", "parking_lot", "fs"] }
 tokio-rustls = { version = "0.24", features = ["dangerous_configuration", "tls12"] }
-reqwest = { version = "0.11", default-features = false, features = ["rustls-tls", "json"] } # TODO: directly use hyper in subscriber module
+reqwest = { version = "0.11", default-features = false, features = ["rustls-tls-native-roots", "json"] } # TODO: directly use hyper in subscriber module
```

Comment:
> One last thing I want to make sure is that by switching to rustls for reqwest that we still validate against the system CA bundle. I recall it was now an option for rustls, but can you make sure we've enabled it properly?

We need to enable the `rustls-tls-native-roots`. I pushed a patch for that, it should be good to go!